gcloud/bigtable/__init__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Bigtable API package."""
+
+try:
+    from grpc._adapter import _c
+except ImportError as exc:
```

Comment:
A much more nuanced argument than "I don't want to" :grinning:

I'm happy to do the pragma no cover (or use the fancy trick I learned from @jonparrott to combine coverage from separate test runs into a single report)